README.md

```diff
@@ -204,7 +204,9 @@ This will generate an HTML file at `htmlcov/index.html` to view code coverage
 
 ### WeasyPrint PDF Library
 
-We are using the [WeasyPrint](https://doc.courtbouillon.org/weasyprint/stable/) library to handle PDF generation on the backend. If you are using the provided Docker images, the dependencies are bundled in the Dockerfile.
+We used the [WeasyPrint](https://doc.courtbouillon.org/weasyprint/stable/) library to handle PDF generation on the backend, but moved to producing PDFs via [LibreOffice](https://github.com/nationalarchives/ds-caselaw-pdf-conversion). But WeasyPrint is still used as a fallback if the PDF is not found in the S3 bucket.
```

Comment:
Worth renaming this section PDF Generation instead?

Reply:
👍🏻